routing/world_graph.hpp

```diff
@@ -67,10 +65,14 @@ class WorldGraph
   virtual RouteWeight HeuristicCostEstimate(Segment const & from, Segment const & to) = 0;
   virtual RouteWeight HeuristicCostEstimate(m2::PointD const & from, m2::PointD const & to) = 0;
   virtual RouteWeight CalcSegmentWeight(Segment const & segment) = 0;
+  virtual RouteWeight CalcLeapWeight(m2::PointD const & from, m2::PointD const & to) const = 0;
   virtual RouteWeight CalcOffroadWeight(m2::PointD const & from, m2::PointD const & to) const = 0;
   virtual bool LeapIsAllowed(NumMwmId mwmId) const = 0;
 
-  // Returns transit-specific information for segment. For nontransit segments returns nullptr.
+  /// @returns transitions for mwm with id |numMwmId|.
+  virtual std::vector<Segment> const & GetTransitions(NumMwmId numMwmId, bool isEnter) = 0;
+
+  /// @returns transit-specific information for segment. For nontransit segments returns nullptr.
```

Comment:
As far as I remember I asked to to put `@returns`. But taking into account that we use an other doxygen notation let's exchange `@returns` with `\returns` for uniformity.

Reply:
done